stix_shifter/stix_translation/src/modules/guardium/json/ISC-Work.code-workspace

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You'll need to remove this file, it's a VS Code file for your workspace.

Reply:
done

Reply:
Removed.